homeassistant/components/alexa/capabilities.py

```diff
@@ -716,3 +716,10 @@ def configuration(self):
         if code_format == FORMAT_NUMBER:
             return {"supportedAuthorizationTypes": [{"type": "FOUR_DIGIT_PIN"}]}
         return []
+
+
+class AlexaChannelController(AlexaCapibility):
+    """Implements Alexa.ChannelController. https://developer.amazon.com/docs/device-apis/alexa-channelcontroller.html """
+
+    def name(self):
+        return 'Alexa.ChannelController'
```

Comment:
```suggestion
        return 'Alexa.ChannelController'
     
```
Need newline at end of file.